### PR TITLE
feat(MOPLAT-379): Add a dev menu to folio

### PR DESCRIPTION
**Pull request by patrinoua**
### Description

This PR resolves [MOPLAT-379]

For now just adds a menu option that says hello.

Is something like that enough or we would need a more elaborate solution?

Screen Shot 2022-08-23 at 15 26 22

### PR Checklist
- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
### To the reviewers 👀
- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-379]: https://artsyproduct.atlassian.net/browse/MOPLAT-379?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by admbtlr** [comment]
Oh nice, I never saw this library before. This is actually probably enough - I guess we can add a "Change environment" entry that switches between Staging and Production and then logs you out.

File: .tool-versions [hunk 1]
```diff
@@ -1,2 +1,2 @@
-ruby 2.7.6
+ruby 2.7.2
```
Comment (admbtlr) [inline .tool-versions:1]: Uh weird - how come Ruby got downgraded?

**Comment by MounirDhahri** [comment]
Same here, first time I see this library. It's also quite small https://github.com/zoontek/react-native-dev-menu/blob/master/src/index.ts

**Comment by patrinoua** [comment]
@admbtlr @MounirDhahri I also hadn't seen it before but seems to work!

**Comment by patrinoua** [comment]
replaced by #175

**Pull request closed without merging**